### PR TITLE
Updated nest to v8 and (almost) all packages

### PR DESCRIPTION
**Pull request by valentinyanakiev**
Apollo express and Ory Client are pending. Ory requires update on the server and Apollo requires careful planning as subscriptions, upload, graphql playground are handled differently in v3 than v2.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,91 +36,73 @@
     "migration:show": "npm run typeorm migration:show"
   },
   "dependencies": {
-    "@jolocom/sdk": "^1.2.0",
```
Comment (techsmyth) [inline package.json:39]: Please leave the jolocom dependencies in

File: package.json [hunk 2]
```diff
@@ -36,91 +36,73 @@
...
     "passport-jwt": "^4.0.0",
     "reflect-metadata": "^0.1.13",
     "replace-special-characters": "^1.2.5",
     "rimraf": "^3.0.2",
-    "rxjs": "^6.6.7",
+    "rxjs": "^7.3.0",
     "sqlite3": "^5.0.2",
-    "typeorm": "0.2.32",
-    "uuid": "^8.3.2",
+    "typeorm": "^0.2.35",
```
Comment (techsmyth) [inline package.json:74]: Interesting if this is now working, would imply that the bug was triggered by one of the dependencies that were removed...

**Review by techsmyth — APPROVED** [review]
Looks like a good tidy up. Have not tried running it yet.

Key to capture the other notes re Apollo version / Ory update elsewhere.

File: package.json (hunk 1 above)
Comment (valentinyanakiev) [reply package.json:39]: I don't think we should until they fix their issues and update their packages. They haven't updated multiple packages in ages (e.g. typeorm dependency 0.2.24 is causing issues with our upgrade, and current typeorm version is 0.2.35, which is conflicting to theirs). Running dependency analyzer shows we are not actively using it either so it's just a nuisance - I suggest we re-add it once it is ready to be used.

File: package.json (hunk 2 above)
Comment (valentinyanakiev) [reply package.json:74]: Which bug and how can I check? I recall there was a bug but I haven't tested this update against it...

File: package.json (hunk 1 above)
Comment (techsmyth) [reply package.json:39]: ok, lets not hold up this PR then - can you please raise another issue with the problems you ran into when using their components with latest version of typeorm?

File: package.json (hunk 2 above)
Comment (techsmyth) [reply package.json:74]: https://github.com/alkem-io/server/issues/1088

Comment (valentinyanakiev) [reply package.json:74]: Haven't seen that error running the application, merging the PR

**Comment by Comoque1** [comment]
Tests are passing against this branch, so suggest to merge it.

**Pull request merged**